integration/configs/browserCompatibilityTest/desktop/linux/content_share_video_test.config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "`": 1,
+  "name": "content_share_video_test.config.json %ts",
+  "grids": [],
+  "tests": [
+    {
+      "name": "Content Share Video Test %ts",
+      "description": "Video sharing using content share",
+      "tupleSize": 1,
+      "noOfThreads": 1,
+      "testImpl": "ContentShareVideoTest.js",
+      "payload": {
+        "url": "http://localhost:8080/",
+        "retry": 2
+      }
+    }
+  ],
+  "clients": [
```

Comment:
For browser compatibility do we want to have other browsers considering canaries run in Chrome. They are mac latest I think but here do we need FF, Safari as well?

Reply:
We already test it in Mac and Windows, this config is only for Linux so Safari probably doesn't make sense :)